spec/services/allocations/finder_spec.rb

```diff
@@ -144,8 +143,8 @@
       context 'with nil status' do
         let(:filter_params) { { status: nil } }
 
-        it 'returns only allocations without a status' do
-          expect(allocation_finder.call).to contain_exactly(allocation)
+        it 'returns empty results set' do
```

Comment:
Just to double check this has been coordinated with the FE to make sure they pass status filters correctly just incase

Reply:
This is just updating the spec to reflect the fact that we don't have any allocations with a `nil` status - if the front end is passing this filter as `nil` currently then no allocations would be returned, so it's safe to conclude this filter is either being omitted when not required.